public/app/features/canvas/elements/text.tsx

```diff
@@ -158,6 +159,8 @@ export const textItem: CanvasElementItem<TextConfig, TextData> = {
       data.color = ctx.getColor(cfg.color).value();
     }
 
+    setDataLinks(ctx, cfg, data);
```

Comment:
rather than having this as a mutating function... what about something like this:
```suggestion
    data.links = findDataLinks(ctx, cfg, data);
```

Reply:
updated. also filtered out the "hide from tooltip" fields.